commit.c

```diff
@@ -358,7 +358,12 @@ void release_commit_memory(struct commit *c)
 	free_commit_buffer(c);
 	free_commit_list(c->parents);
 	/* TODO: what about commit->util? */
+	/*
+	It seems as if we can remove this todo. util seems to not be a field of the struct commit,
+	definition of struct commit --> commit.c:27
+	The commit list also, does not contain a field for util.
 
+	*/
```

Comment:
Good point.

Let's just remove the line? Maybe you can identify the commit that forgot to remove it? (Hint: `git log -L <argument>` is your friend...)

Once you identify the commit, you could imitate the commit message of 60129c61e55fa5567d8b7ebe21b607e363330f07:

- the lines wrap at 76 columns/line
- the first line is a concise title
- the commit message gives a compelling reason for the change, answering the "why" (the patch already explains the "how")

Reply:
Sounds good. Will do asap.